include/cmocka.h

```diff
@@ -160,6 +160,14 @@ cast_to_largest_integral_type(cast_to_pointer_integral_type(value))
 #define CMOCKA_PRINTF_ATTRIBUTE(a,b)
 #endif /* __GNUC__ */
 
+#ifdef __GNUC__
+#define SOF_CMOCKA_NORETURN __attribute__ ((noreturn))
```

Comment:
I know this is in `thesofproject`, but there is no need to add `SOF_` suffiix to names, this repo can be considered as just optimized/fixed for embedded/xtensa, not something *just for SOF*.

Reply:
I did this to highlight this change was not upstream but fair enough, that information is already in `tig blame`. Will remove.